src/gui/command.ts

```diff
@@ -120,36 +120,36 @@ export class FindElementsFileCommand {
                 workspace.workspaceFolders?.map(({ uri }) => uri.fsPath),
             );
         } catch (error: any) {
-            getLog().info(l10n.t("Find element descriptors file: "), error.message || error);
+            getLog().info(l10n.t("Find element descriptor files: "), error.message || error);
         }
         pythonPath = pythonPath || workspace.getConfiguration("python").get("defaultInterpreterPath", "python");
         window.withProgress(
             {
                 location: ProgressLocation.Notification,
                 cancellable: false,
-                title: l10n.t("Finding visual element descriptors file"),
+                title: l10n.t("Finding visual element descriptor files"),
             },
             async (progress) => {
```

Comment:
Is this 'progress' used anywhere?

Reply:
it is not, just the default parameter of the returned function, we can make it private